test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -280,7 +286,14 @@
           [foo-service baz-service fail-service]
           (let [resp (http-client/get "http://localhost:8180/status/v1/simple")]
             (is (= 503 (:status resp)))
-            (is (= "error" (slurp (:body resp)))))))
+            (is (= "unknown" (slurp (:body resp)))))))
```

Comment:
Hmm. I see how this happened now. The docstring does specify that `unknown` should be returned whenever the states don't all agree. The wire format documentation doesn't exactly specify the behavior -- it says `unknown` is used when there's a problem getting the status for some service, but doesn't say any more than that.

I think that the existing code captures the correct behavior, so the docstring should be updated. I think it's most useful to surface the "most severe" state that we got: if any one state is `error`, then that's what we should return; if none are `error`, then if any are `unknown` that gets returned; finally if all states are `running` then that's what we return. Equivalently, the states have a severity ranking and we return the highest-severity state that we got according to that ranking.

I'm not sure where `starting` and `stopping` fit into that ranking, but my initial impulse is that the ordering should be (from most to least severe):
* error
* unknown
* stopping
* starting
* running

@wkalt I am sorry for piling on all the questions under the assumption you wanted to change this behavior. I see now you were trying to match what you believed the correct behavior to be. I should have been less presumptive.